CLAUDE.md

```
@@ -211,3 +211,16 @@ ButtonPrimaryYellow(
 
 - Unit tests in `app/src/test/`
 - Instrumentation tests in `app/src/androidTest/`
+
+## Git Commit Messages
+
+Use **plain imperative mood**, no type prefixes (`feat:`, `chore:`, etc.):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)